### DISCUSSION TITLE
Does this violate Kurento's TOS?

### DISCUSSION THREAD

**Discussion by neilyoung** [Kurento/kurento#20] [Community]
Customer asked me to change the `s=Kurento Media Server` to something customer specific in an OFFER generated by KMS. Is that agains't some TOS?

**Comment by j1elo** [comment]
Absolutely not. You can do basically anything that the Apache 2.0 license allows you to do; changing the code and redistributing your own build is among those things.

**Comment by neilyoung** [comment]
Thanks for clarification
